langs/go.go

```diff
@@ -125,6 +126,7 @@ func myHandler(ctx context.Context, in io.Reader, out io.Writer) {
 	}{
 		Msg: fmt.Sprintf("Hello %s", p.Name),
 	}
+	log.Print("This is a sample log message.")
```

Comment:
Just a nit: rather than logging `This is a sample log message.`, it could be `invoking function (Hello, world!)`, which (to me) looks more like auto-generated boilerplate and less like sample code.

Reply:
Will make this change now.

Reply:
Done.